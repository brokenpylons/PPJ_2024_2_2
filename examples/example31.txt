(60 + 89) / 2 ^ 2
