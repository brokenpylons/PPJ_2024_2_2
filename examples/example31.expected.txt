lparen("(") real("60") plus("+") real("89") rparen(")") divides("/") real("2") pow("^") real("2") 
